deploy/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
nu deze file er is kunnen we deploy/docker-wait.sh verwijderen.

Reply:
check, goeie